neqo-crypto/src/agent.rs

```diff
@@ -414,10 +414,6 @@ impl SecretAgentInfo {
     pub const fn alpn(&self) -> Option<&String> {
         self.alpn.as_ref()
     }
-    #[must_use]
-    pub const fn signature_scheme(&self) -> SignatureScheme {
```

Comment:
Huh, I'd have thought that we'd want to have metrics for that in Firefox.  Oh well.